lib/utils/group-helper.js

```diff
@@ -1,6 +1,22 @@
 const { resolve, join } = require('path');
 const { existsSync } = require('fs');
 
+const parseAndValidatedFilename = pathToResolve => {
+    const pathToArray = pathToResolve.split('/');
+    let pathBeforeFileName;
+    let fileName = pathToArray[pathToArray.length - 1];
+    const isLastNameAFile = existsSync(resolve(join(pathToArray.slice(0, pathToArray.length - 1).join('/'), fileName.includes('.js') ? fileName : fileName + '.js')));
+    fileName = isLastNameAFile ? fileName : 'index.js';
+    fileName = fileName.includes('.js') ? fileName : fileName + '.js';
```

Comment:
This could be turned into a helper method as it is being reused.

Reply:
May be! I guess we can export it from here itself. what do you think! @evenstensberg what do you think of adding it to the separate utils file or exporting from here itself?